lib/ex_huobi/futures/rest/account.ex

```diff
@@ -9,11 +9,11 @@ defmodule ExHuobi.Futures.Rest.Account do
 
   @hbdm_host "https://api.hbdm.com"
 
-  @spec get_position(String.t(), config) :: {:error, any} | {:ok, any}
-  def get_position(instrument_id, config) do
+  @spec get_positions(String.t(), config) :: {:error, any} | {:ok, any}
+  def get_positions(instrument_id, config) do
```

Comment:
Why not still keep the old endpoint and add a new one ? @quantd2

Reply:
not like okex, the huobi one returns separated positions: sell and buy, so I marked it plural.